docs/Troubleshooting.md

```diff
@@ -92,29 +92,6 @@ Edit `config/initializer/shopify_app.rb` and ensure the following configurations
 + config.shop_session_repository = 'Shop'
 ```
 
-#### Inspect server logs
```

Comment:
I don't know if we want to remove all of this, are we sure these logs are irrelevant now?